vercel.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
taken from https://github.com/across-protocol/frontend-v2/blob/master/vercel.json

Reply:
@evaldofelipe is this all good?

Reply:
yes sir! good to go!